content/features/content-management/recycle-bin/index.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: recycle-bin
+locale: en
+title: Recycle Bin
+dnnversion: 09.10.00
+---
+# Recycle Bin
+# Overview
+Just like your computer, when you delete content from your website it does not instantly get deleted from the system. Instead it is sent to a recycling bin that can be found under `Content` - `Recycle Bin`. There you will be able to view, restore, or permanently delete content that has been sent to the bin. If you want to empty your recycling bin, you can press the `Empty Recycle Bin` button. 
+
+![RecycleBin](https://user-images.githubusercontent.com/44900498/157530166-0652335d-560c-43eb-b94e-342ce46d1410.PNG)
```

Comment:
Images need to but part of the repository and not be github user images

Reply:
Okay, I just committed the fix for this one and the commits at the top seem to show it working. Thanks for pointing this out.